db/app.test.js

```diff
@@ -72,7 +71,33 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/wnfjn134")
       .expect(400)
       .then(({ body }) => {
-        expect(body.message).toBe(`Bad request`);
+        expect(body.message).toBe("Invalid Article ID !");
+      });
+  });
+});
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200 responds with the object of all comments from the article with the path of the article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("votes", expect.any(Number));
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("article_id", expect.any(Number));
+          expect(comment).toHaveProperty("created_at", expect.any(String));
+        });
+        expect(comments).toBeSortedBy("created_at", { descending: true });
```

Comment:
These assertions look great, but what would happen if `comments` was an empty array? It's important to make sure our test avoids false positives.